tests/functional/test_install_reqs.py

```diff
@@ -191,6 +191,42 @@ def test_install_local_editable_with_subdirectory(script):
     result.assert_installed('version-subpkg', sub_dir='version_subdir')
 
 
+def test_user_with_prefix_in_pydistutils_cfg(script, data, virtualenv):
+    """
```

Comment:
Hurts my brain too. I was copying the style of test_install_option_in_requirements file, but now I see that most tests in this file have no docstring.

I've removed both of those and the errant whitespace.